roles/etcd/tasks/etcdctl.yml

```diff
@@ -4,9 +4,9 @@
   when: not openshift.common.is_atomic | bool
 
 - name: Configure etcd profile.d alises
-  copy:
+  template:
```

Comment:
Can't template from a shell script.  The source file needs to be updated to a jinja template if that was the intention.